webapp/app.py

```diff
@@ -63,7 +77,7 @@ def results():
 
 
 @app.route(
-    "/careers/<regex('[a-z-]*[a-z][a-z-]*'):department>",
+    "/careers/<any('({})'):department>".format(str(departments)[1:-1]),
```

Comment:
IMO it would be easier do this in the view, something like:
```
if department not in departments:
   flask.abort(404)
```
@nottrobin @tbille?

Reply:
`@app.route("/careers/<department>")`